app/build.gradle

```diff
@@ -67,4 +69,19 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+}
+
+//It will create a new task(s) based on your application variants within (run) group
+//sample: ./gradlew installRunDebug
+//sample: ./gradlew installRunStagDebug
+project.afterEvaluate {
+    android.applicationVariants.all { variant ->
+        task "installRun${variant.name.capitalize()}"(type: Exec, dependsOn: "install${variant.name.capitalize()}", group: "run") {
```

Comment:
we can access the install task via `variant.install`, no need to guess the name directly.